nemo/collections/asr/parts/utils/vad_utils.py

```diff
@@ -1393,16 +1457,24 @@ def get_frame_labels(segments: List[List[float]], frame_length: float, offset: f
 
 
 def plot_sample_from_rttm(
-    audio_file: str, rttm_file: str, max_duration: Optional[float] = None, save_path: str = "", show: bool = True
+    audio_file: str,
+    rttm_file: str,
+    max_duration: Optional[float] = None,
+    save_path: str = "",
+    show: bool = True,
+    offset: float = 0.0,
+    unit_frame_len: float = 0.01,
 ):
+    """
+    Plot audio signal and frame-level labels from RTTM file
+    """
     plt.figure(figsize=[20, 2])
-    UNIT_FRAME_LEN = 0.01
 
-    audio, sample_rate = librosa.load(path=audio_file, sr=16000, mono=True, offset=0, duration=max_duration)
+    audio, sample_rate = librosa.load(path=audio_file, sr=16000, mono=True, offset=offset, duration=max_duration)
```

Comment:
Please see [Comment ID: a03 - Please change these lines to use `AudioSegment.from_file()`]